assets/src/legacy/attributeTable.js

```diff
@@ -2207,6 +2207,7 @@ var lizAttributeTable = function() {
                 var layerConfig = config.layers[typeName];
                 layerConfig['features'] = {};
                 var foundFeatures = {};
+                const wmsName = lizMap.mainLizmap.state.layersAndGroupsCollection.getLayerByName(layerConfig.name).wmsName;
```

Comment:
```suggestion
                const wmsName = layerConfig?.shortname || layerConfig.name;
```
To keep compatibility with legacy code and to be sure to backport automatically.